src/index.js

```diff
@@ -33,8 +26,39 @@ items.forEach((item) => {
 
   input.addEventListener('change', () => {
     updateCompleted(item, input);
-    saveLocalStorage(items);
+    saveLocalStorage(itemsArray);
   });
 
   todos.appendChild(div);
+  const vertical = div.querySelector('.vertical');
+  const trash = div.querySelector('.trash');
+  const edit = div.querySelector('.edit');
+  edit.addEventListener('change', () => {
+    editTodo(itemsArray, div.id, edit.value);
+    saveLocalStorage(itemsArray);
+  });
+  vertical.addEventListener('click', () => {
+    vertical.style.display = 'none';
+    trash.style.display = 'block';
+  });
+  trash.addEventListener('click', () => {
+    div.remove();
+    itemsArray = itemsArray.filter((i) => removeTodos(i, div.id));
+    saveLocalStorage(itemsArray);
+  });
+});
+
+const form = document.getElementById('form-id');
```

Comment:
- [ ] There is an issue with the "add" functionality because I was not able to add elements, the console throws an error about `null` for this item and it is because you have no element with "form-id" :pushpin:
![image](https://user-images.githubusercontent.com/81522435/145215351-c3110848-3ad2-4998-be42-24e44d622e97.png)
I tried fixing this by adding the `form-id` value to an element :wink: